kubernetes/samples/charts/nginx/samples/tls.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+apiVersion: voyager.appscode.com/v1beta1
```

Comment:
change it to:
apiVersion: networking.k8s.io/v1beta1

Reply:
done